app/assets/javascripts/modules/OffenderRejectedDynamicTextArea.js

```diff
@@ -0,0 +1,22 @@
+moj.Modules.OffenderRejectedDynamicTextArea = {
+  // Offender SAR reason-rejected cases partial
+  // Provides a dynamic text_area which displays when the user chooses the 'other' checkbox
+  init: function () {
+    const other_checkbox = document.getElementById('offender_sar_rejected_reasons_other');
+    const text_area = document.querySelector('div.display-none');
+
+    if (!other_checkbox) return;
```

Comment:
I think it's worth adding a blank line after to make this stand out more
```suggestion
    if (!other_checkbox) return;
    
```